testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -139,7 +140,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
      * Change this whenever you add or remove subprojects for distribution-packaged plugins (lib/plugins).
      */
     int getPackagedPluginsJarCount() {
-        82
+        81
```

Comment:
🎉 Happy to see this number going down!
❌ OTOH, make sure that this won't cause a problem: https://github.com/gradle/gradle/blob/465dd41bcb68e6b079b3de151b9ef1dbc3d214c9/platforms/core-runtime/build-process-services/src/main/java/org/gradle/api/internal/DefaultClassPathProvider.java#L81

Reply:
> OTOH, make sure that this won't cause a problem:

Reply:
The number decreased because the `problems-rendering` subproject was part of the plugin jars. This is because it was a transitive dependency of the `java` plugin. Because `problems-rendering` now used by core, it's now moved to a lib dependency.

Reply:
SGTM, thanks for having a look.